fga-eps-mds/2020.1-stay-safe-user-service
# Increase coverage

## Issue 
Resolves #46 


## Description
Add more tests and cases of tests.

## Checklist:



## How to validate?
Run the tests with
```bash
$ docker-compose run api coverage run -m pytest
```
And check the coverage with command
```bash
$ docker-compose run api coverage report -m
```
